assets/admin/package.json

```diff
@@ -8,6 +8,7 @@
         "watch": "webpack --mode development -w"
     },
     "dependencies": {
+        "@ckeditor/ckeditor5-font": "^18.0.0",
```

Comment:
It is important that the plugin version is compatible with the ckeditor version used in the project. The ckeditor version of the project can be outputted by executing `npm list @ckeditor/ckeditor5-core`.